kura/org.eclipse.kura.linux.bluetooth/META-INF/MANIFEST.MF

```diff
@@ -2,11 +2,12 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: org.eclipse.kura.linux.bluetooth
 Bundle-SymbolicName: org.eclipse.kura.linux.bluetooth;singleton:=true
-Bundle-Version: 1.0.4
+Bundle-Version: 1.0.5
 Bundle-RequiredExecutionEnvironment: JavaSE-1.6
 Import-Package: org.apache.commons.io;version="2.4.0",
  org.eclipse.kura;version="[1.0,2.0)",
- org.eclipse.kura.bluetooth;version="[1.2.0,1.3.0)",
+ org.eclipse.kura.bluetooth;version="[1.3.0,1.4.0)",
+ org.eclipse.kura.bluetooth.listener;version="[1.0,1.1.0)",
```

Comment:
1. We are the consumer of ``org.eclipse.kura.bluetooth.listener``. import with ``[1.0,2.0)``
2. Import ``org.eclipse.kura.bluetooth`` omitting micro version, i.e. ``[1.3,1.4)``